### PR TITLE
Collaborate 2.0 features

### PR DESCRIPTION
**Pull request by divyabhushan**
/assign @vbisrikkanth

### CODE REVIEW

File: docs/intro/introduction.mdx [hunk 1]
```diff
@@ -34,7 +35,7 @@ arrive at actionable decisions. ValQ supports use cases such as what-if simulati
 
 <div style={{textAlign: 'center'}}>
   <Zoom>
-    <img alt="ValQ uses" src={useBaseUrl('/doc-images/intro/2.png')} 
+    <img alt="ValQ used cases" src={useBaseUrl('/doc-images/intro/2.png')}
```
Comment (vbisrikkanth) [inline docs/intro/introduction.mdx:38]: Typo fix  use cases

File: docs/releases.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+---
```
Comment (vbisrikkanth) [inline docs/releases.md:1]: We're not adding this page right?

File: docs/storage/features/commenting.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import { Tag } from "../../../src/ProductTag.js";
+
+<Tag
+header="Commenting"
+image={useBaseUrl("/doc-images/valq-collaborate.png")}
+/>
+
+The model collaboration feature allows the members added to a model to view, share, and comment on each other's changes.
+The *Model creator* must <Link to={useBaseUrl("docs/storage/features/save-model")}>Save and upload</Link> the model before collaborating with other users.
+In addition, the model must also be shared with users/groups with a dedicated <Link to={useBaseUrl('#')}>user role</Link>.
```
Comment (vbisrikkanth) [inline docs/storage/features/commenting.md:21]: We can remove "with a dedicated <Link to={useBaseUrl('#')}>user role</Link>"

File: docs/storage/features/commenting.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+<div style={{ textAlign: "center" }}>
+  <Zoom>
+    <img alt="Node commenting" src={useBaseUrl("/doc-images/storage/features/planning-series-add-comments.png")} />
+  </Zoom>
+</div>
+
+*Comment on the selected node*
+
+### Add comments for Simulation
```
Comment (vbisrikkanth) [inline docs/storage/features/commenting.md:57]: Change Simulation to Scenarios

File: docs/storage/features/export-old.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
+---
```
Comment (vbisrikkanth) [inline docs/storage/features/export-old.md:1]: Why do we have export-old and export?

File: docs/storage/features/model-merge-old.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
+---
```
Comment (vbisrikkanth) [inline docs/storage/features/model-merge-old.md:1]: Why do we have model merge old document as well?

File: docs/storage/read_mode.mdx [hunk 7]
```diff
@@ -0,0 +1,153 @@
+---
```
Comment (vbisrikkanth) [inline docs/storage/read_mode.mdx:1]: We can remove this page from the documentation.

File: src/ProductRelease.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (vbisrikkanth) [inline src/ProductRelease.js:1]: Are we using this?

**Review by vbisrikkanth — CHANGES_REQUESTED** [review]

File: docs/releases.md (hunk 2 above)
Comment (divyabhushan) [reply docs/releases.md:1]: yes, it's disabled in sidebars.js. Will also delete in prod.

File: src/ProductRelease.js (hunk 8 above)
Comment (divyabhushan) [reply src/ProductRelease.js:1]: yes

**Pull request merged**